pyk/src/pyk/kast/formatter.py

```diff
@@ -76,6 +77,10 @@ def format(self, term: KInner) -> str:
             term = add_brackets(self.definition, term)
         return ''.join(self._format(term))
 
+    @property
+    def printer(self) -> Callable[[KInner], list[str]]:
+        return lambda k: self.format(k).split('\n')
```

Comment:
This feels like an implicit dependency in the wrong direction: this method is only introduced so that `CTermShow` can be instantiated with a `Fomatter`.

Instead, consider the `@staticmethod` approach, which makes the dependency between these two classes explicit.

Reply:
I removed this.